app/model/Skautis/ParticipantRepository.php

```diff
@@ -220,9 +220,11 @@ private function processParticipants(array $participantsSis, Event $event): arra
             $participants[$p->ID] = ParticipantFactory::create($p, $payment);
         }
 
+        /*
```

Comment:
Tady moc nevím, co s tím - protože je obdobný kód i v `InstrucotRepository` a oba šahají do stejné tabulky, musel jsem tohle zakomentovat - když jsem to neudělal, tak si `ParticipantRepository` a `InstructorRepository` vzájemně mazaly platby.

Asi nejčistší řešení by bylo instruktorské platby v databázi mít v jiné tabulce, ale to už je dost výrazná úprava a moc se mi do toho nechce. A vlastně nevím, k čemu tady tenhle kód je, jestli je vůbec potřeba. Napadá mě něco jako odebírání úćastníků, kteří mají vyplněné poplatky, ale to by se podle mě mělo spíš smazat při tom odebrání - tady se to maže až při dalším načtení seznamu účastníků...

Reply:
kdyz nekdo smaze ucastnika pres skautis, tak nam by tu zustala vyset platba bez ucastnika. A tohle to je mechanismus jak to smazat.